shared/views/FilterCardView.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const { TemplateView, AvatarView } = require('hydrogen-view-sdk');
+const AvatarViewModel = require('../viewmodels/AvatarViewModel');
+
+class FilterCardView extends TemplateView {
```

Comment:
Would it be easier to combine this logic into `RoomCardView`?

I'm not totally opposed to having something separate but feels like could be combined